vmdb/app/models/vm_or_template/operations/smart_state.rb

```diff
@@ -0,0 +1,9 @@
+class VmOrTemplate
+  module Operations
+    module SmartState
+      def validate_scan
+        {:available => true, :message => nil}
```

Comment:
We can keep this as interim code, and then in a follow up PR do the proper breakdown by provider.  The UI code should probably use this same validator code, instead of custom logic in the controller... cc @dclarizio